File: src/main/java/no/nav/bidrag/aktoerregister/persistence/repository/HendelseRepository.java
package no.nav.bidrag.aktoerregister.persistence.repository;

import java.util.List;
import no.nav.bidrag.aktoerregister.persistence.entities.Aktoer;
import no.nav.bidrag.aktoerregister.persistence.entities.Hendelse;

public interface HendelseRepository {

  List<Hendelse> hentHendelser(int fraSekvensnummer, int antallHendelser);

  void opprettHendelser(List<Aktoer> updatedAktoerer);
}

File: src/main/kotlin/no/nav/bidrag/aktoerregister/persistence/repository/HendelseRepository.kt
package no.nav.bidrag.aktoerregister.persistence.repository

import no.nav.bidrag.aktoerregister.persistence.entities.Aktør
import no.nav.bidrag.aktoerregister.persistence.entities.Hendelse

interface HendelseRepository {
    fun hentHendelser(fraSekvensnummer: Int, antallHendelser: Int): List<Hendelse>
    fun opprettHendelser(updatedAktoerer: List<Aktør>)
}
